### ISSUE TITLE
1.18.1

### ISSUE THREAD

**Issue by CharlesNic** [eccentricdevotion/TARDIS#590]
Long no see!   I recently decided to start testing 1.18.1 with partial clone of my server.  I've been updating and testing the plugins I use.    My server is so huge and the plugins make it a very long process, until everything is working happy.   

I'm in rush for anything, but thought I'd check to see if you had eventual plans to update Tardis.

Thank you and Happy New Year!

**Comment by Chew** [comment]
TARDIS fully supports 1.18.1 as of https://github.com/eccentricdevotion/TARDIS/commit/4af8ce5b84d4c67a7c16e79fe7ed53abffdc0f84

All latest builds should be fully supported!

**Comment by CharlesNic** [comment]
ok :)   I guess I need to do a little more troubleshooting this weekend.  It throws errors when Paper loads it.

**Comment by CharlesNic** [comment]
I decided to remote into my home PC and striped out most of the plugins, except multiverse, LuckPerms, DeathCoordinates, Essentials-X, Vault and worldguard.    I hadn't noticed that PreciousStones was failing to load.

As soon as I removed PreciousStones, Tardis loaded. I had a lot of worlds not imported into my server clone and it started generated new ones.  lol.   Needless to say, I halted the server.   I may copy and migrate my other worlds over this week, but I'm sure it will work now.

Thank you!
